### PR TITLE
dotcom: Improve copy & styling of 3rd step in welcome flow

### PR DESCRIPTION
**Pull request by kopancek**
Closing in favor of #23986

### CODE REVIEW

**Review by artemruts — APPROVED** [review]
✨

File: client/web/src/auth/welcome/StartSearching.tsx [hunk 1]
```diff
@@ -129,15 +129,15 @@ export const StartSearching: React.FunctionComponent<StartSearching> = ({
 
     return (
         <div className="mt-5">
-            <h3>Start searching...</h3>
+            <h4>Fetching repositories...</h4>
```
Comment (quinnkeast) [inline client/web/src/auth/welcome/StartSearching.tsx:132]:
```suggestion
            <h3>Fetching repositories...</h3>
```

Sorry, we got swapped somewhere!

File: client/web/src/auth/welcome/StartSearching.tsx [hunk 2]
```diff
@@ -129,15 +129,15 @@ export const StartSearching: React.FunctionComponent<StartSearching> = ({
...
             <p className="text-muted mb-4">
                 We’re cloning your repos to Sourcegraph. In just a few moments, you can make your first search!
             </p>
             <div className="border overflow-hidden rounded">
                 <header>
-                    <div className="py-3 px-4">
-                        <h3 className="d-inline-block m-0">Activity log</h3>
-                        <span className="float-right m-0">{statusSummary}</span>
+                    <div className="py-3 px-4 d-flex justify-content-between align-items-center">
+                        <h3 className="m-0">Activity log</h3>
```
Comment (quinnkeast) [inline client/web/src/auth/welcome/StartSearching.tsx:139]:
```suggestion
                        <h4 className="m-0">Activity log</h4>
```

Comment (quinnkeast) [reply client/web/src/auth/welcome/StartSearching.tsx:139]: Second part of the swap.

**Review by quinnkeast — CHANGES_REQUESTED** [review]
Looks good! Only change is that the `h3` → `h4` swap is in the wrong spot. I've added comments to fix.

**Review by unknwon — APPROVED** [review]

**Review by quinnkeast — APPROVED** [review]

**Pull request closed without merging**